src/Alerts/atom/Notification.tsx

```diff
@@ -0,0 +1,153 @@
+import { resetButtonStyles } from '../../common/index';
+
+const closeAnimation = keyframes`
+  0% {
+    transform: translate(-50%, 0);
+  }
+  100% {
+    transform: translate(-50%, -100%);
+  }
+`;
+
+const Container = styled.div<{type: AlertType, isClosing: Boolean}>`
+  min-height: 50px;
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 14px;
+  max-width: 900px;
+  position: fixed;  
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 999;
+
+  font-family: ${({ theme }) => theme.fontFamily.ui };
+  ${({type, theme}) => theme.styles.feedbackBar[type] };
+  ${({theme}) => theme.typography.feedbackBar.message };
+
+  ${({theme, isClosing}) => isClosing && css`
+    animation: ${closeAnimation} ${theme.animation.speed.slower} ${theme.animation.easing.primary.easeInOut};
+    `
+  };
+
+`;
+
+export const IconNames = {
+  error: 'Critical',
+  warning: 'BigWarning',
+  success: 'Success',
+  info: 'Information',
+  neutral: 'Information'
+}
+
+const IconButton = styled.div<{selected?: boolean}>`
+  ${resetButtonStyles};
+  ${({selected=false}) => selected && css`
+    border-bottom: 5px solid hsl(207, 80%, 64%);
+  `}
+  &:focus {
+    outline: none;
+  }
+ 
+  &:hover {
+    opacity: .8;
+  }
+  &:active:enabled {
+    opacity: .9;
+  }
+  &:disabled {
+    opacity: 0.1;
+  }
+`;
+
+const TextButton = styled.button`
+  ${resetButtonStyles};
+  font-family: ${({ theme }) => theme.fontFamily.ui };
+  ${({theme}) => theme.typography.feedbackBar.message };
+  font-weight: 700;
+
+  &:focus {
+    outline: none;
+  }
+
+  &:hover:enabled {
+    ${({theme}) => theme && css`
+      opacity: .8;
+      transition: transform ${theme.animation.speed.normal} ${theme.animation.easing.primary.inOut};
+    `}
+  }
+  &:active:enabled {
+    opacity: .9;
+  }
+  &:disabled {
+    opacity: 0.1;
+  }
+`;
+
+const MainMessage = styled.div`
+  padding: 12px;
+`;
+
+interface Props {
+  type?: AlertType
+  message?: string;
+  autoDismiss?: boolean;
+  actionTextButton?: string;
+  actionHandler?: () => void;
+}
+
+const Notification : React.FC<Props> = ({type ='info', message, autoDismiss = false, actionTextButton, actionHandler}) => {
+  const [dismiss, setDismiss] = useState(false);
+  const [slideUp, setSlideUp] = useState(false);
+
+  useEffect(()=>{
+    setDismiss(false);
+    setSlideUp(false);
+  },[message]);
+
+  const handleDismiss = useCallback(() => {
+
+    if(actionHandler) {
+      actionHandler();
+    }
+    setSlideUp(true);
+  },[actionHandler])
+
+  const animationEndTest = () => {
+    if(slideUp){
+      setDismiss(true);
+    }
+  }
+
+  // https://ux.stackexchange.com/questions/85882/for-how-long-should-alerts-be-displayed
+  useEffect(() => {
+    if(autoDismiss) {
+      setTimeout( () => {
+        handleDismiss();
+      }, message ? (message.length * 50) : 1000);
```

Comment:
Having this in mind the article says 2000 as min and 7000 as max. Japanese natives might read faster the Kanji but as I foreigner I feel it would take me a lot.  - https://irisreading.com/average-reading-speed-in-various-languages/

I think it might be better to keep it static, following the suggestion of the max time is 7000.

Reply:
Sounds good, let's do 7000 for now and see how people like it when we start to use it.